ui/src/tempVars/components/EditTemplateVariable.tsx

```diff
@@ -0,0 +1,271 @@
+import React, {
```

Comment:
what do you think of naming this component `TemplateVariableEditor` for consistency with other component names? or some other active name or noun-based function. two name conventions that i'm recalling currently are `{action}or/er` and `{component noun}`, such as `BodyBuilder` and `DashboardPage` or `Dropdown`